src/lib/Microsoft.Fx.Portability.Azure/Storage/StorageBlob.cs

```diff
@@ -0,0 +1,147 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
This library class is missing `.ConfigureAwait(false)` after its await method calls

Reply:
Mostly from old code but fixed!

---
In reply to: [178098712](https://github.com/Microsoft/dotnet-apiport/pull/598#discussion_r178098712) [](ancestors = 178098712)